pkg/util/utils.go

```diff
@@ -542,7 +542,8 @@ func ParseInputTime(inputTime string) (time.Time, error) {
 	if err != nil {
 		return time.Time{}, errors.Errorf("unable to interpret time value")
 	}
-	return time.Now().Add(-duration), nil
+
+	return time.Now().Add(duration), nil //time.Now().Sub(duration)), nil
```

Comment:
This is also used for since, in this case we want it to be the negative duration.